src/core/analyzer-cache-context.ts

```diff
@@ -322,59 +379,7 @@ export class AnalyzerCacheContext {
     return [];
   }
 
-  /**
-   * Scans all of the transitive dependencies of the given document.
-   *
-   * Uses the `visited` set to break cycles.
-   */
-  private async _scanImports(
```

Comment:
Does inlining this into _scan help legibility? I kinda like it more with them separate.